invesalius/data/viewer_volume.py

```diff
@@ -1591,7 +1583,7 @@ def InitializeColorArray(self):
     def ReturnToDefaultColorActor(self):
         self.efield_mesh.GetPointData().SetScalars(self.colors_init)
         wx.CallAfter(Publisher.sendMessage, 'Initialize color array')
-        self.RecolorEfieldActor(self.efield_mesh)
+        self.RecolorEfieldActor()
         wx.CallAfter(Publisher.sendMessage,'Render volume viewer')
 
     def CreateLUTtableforefield(self, min, max):
```

Comment:
Please, update the name of this method to comply with InVesalius convention: **CreateLUTTableForEfield**